tls/s2n_alerts.c

```diff
@@ -222,7 +228,12 @@ int s2n_process_alert_fragment(struct s2n_connection *conn)
                 conn->config->cache_delete(conn, conn->config->cache_delete_data, conn->session_id, conn->session_id_len);
             }
 
-            /* All other alerts are treated as fatal errors */
+            /*
+             *= https://tools.ietf.org/rfc/rfc8446#section-6
+             *# Unknown Alert types MUST be treated as error alerts.
+             *
+             * All other alerts are treated as fatal errors.
+             */
```

Comment:
When a comment comes after the compliance comment I read it as a clarification of the compliance comment. Like "Unknown Alert types MUST be treated as error alerts, and all other alerts are treated as fatal errors." Like "other alerts" sounds like it means "other than unknown alert types". If it comes before it's clearer that's not true.
```suggestion
            /* All other alerts are treated as fatal errors.
             *
             *= https://tools.ietf.org/rfc/rfc8446#section-6
             *# Unknown Alert types MUST be treated as error alerts.
             */
```